.flowconfig

```diff
@@ -0,0 +1,34 @@
+[ignore]
```

Comment:
I've leaned towards putting this file in `src/` (have done that with Relay and Draft) - tends to mean less things to have to ignore and less random breakage when you miss a new addition (eg random npm module)